content/automate/ManageIQ/Infrastructure/VM/Provisioning/StateMachines/VMProvision_VM.class/clone_to_vm.yaml

```diff
@@ -11,4 +11,4 @@ object:
   - PreProvision:
       value: "/Infrastructure/VM/Provisioning/StateMachines/Methods/PreProvision_Clone_to_VM#${/#miq_provision.source.vendor}"
   - EmailOwner:
-      value: "/Infrastructure/VM/Provisioning/Email/MiqProvision_Complete?event=vm_cloned"
+      value: "/System/Notification/Email/InfrastructureMiqProvisionComplete?event=vm_cloned"
```

Comment:
@tinaafitz 
do you know if the event is needed when calling the method, is it used by the called instance or any other child of the called instance.